services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/appconfig/structure/directory/AppConfigurationDirectory.java

```diff
@@ -72,7 +72,8 @@ public AppConfigurationDirectory(CryptoProvider cryptoProvider, DistributionServ
     this.distributionServiceConfig = distributionServiceConfig;
 
     countryDirectory = new IndexDirectoryOnDisk<>(distributionServiceConfig.getApi().getCountryPath(),
-        __ -> Set.of(distributionServiceConfig.getApi().getCountryGermany()), Object::toString);
+        ignoredValue ->
```

Comment:
I guess this one will be a matter of taste but personally I like the suggestion - and it also makes [sonar](https://rules.sonarsource.com/java/RSPEC-115) happy :)

However, there are quite a few more instances of `__ ->` throughout the application that were missed in this PR.

Reply:
I've now changed the remaining occurrences as well.